tools/vsextension/ProjectWizard/WizardImplementation.cs

```diff
@@ -0,0 +1,296 @@
+﻿// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TemplateWizard;
+using System.Windows.Forms;
+using EnvDTE;
+using System.IO;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio;
+
+namespace OpenEnclaveSDK
+{
+    public class WizardImplementation : IWizard
+    {
+        public static string EdlLocation;
+
+        // This method is called before opening any item that   
+        // has the OpenInEditor attribute.  
+        public void BeforeOpeningFile(ProjectItem projectItem)
+        {
+        }
+
+        public void ProjectFinishedGenerating(Project project)
+        {
+        }
+
+        // This method is only called for item templates, not for project templates.  
+        public void ProjectItemFinishedGenerating(ProjectItem projectItem)
+        {
+        }
+
+        // This method is called after the project is created.  
+        public void RunFinished()
+        {
+        }
+
+        // Look in <folder>/conf.mk to see what is supported.
+        private void GetConfFlags(string folder, out bool hardwareFloatSupported, out bool is32Bit)
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            // Set default values.
+            hardwareFloatSupported = false;
+            is32Bit = false;
```

Comment:
There is no support for 32-bit ARM in OE master, unfortunately.

Reply:
Per discussion, this only runs for a Linux project; support for that is coming later.